src/Plugin/jsonapi_hypermedia/LinkProvider/BreadcrumbLinks.php

```diff
@@ -0,0 +1,130 @@
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\ezcontent_api\Breadcrumb\EzContentBreadcrumbBuilder;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Link plugin for article resource.
+ *
+ * @JsonapiHypermediaLinkProvider(
+ *   id = "jsonapi_hypermedia.breadcrumb",
+ *   link_relation_type = "node",
+ *   link_context = {
+ *    "resource_object" = "node--article",
+ *   }
+ * )
+ */
+class BreadcrumbLinks extends LinkProviderBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The plugin_id.
+   *
+   * @var string
+   */
+  protected $pluginId;
+
+  /**
+   * The plugin implementation definition.
+   *
+   * @var array
+   */
+  protected $pluginDefinition;
+
+  /**
+   * Configuration information passed into the plugin.
+   *
+   * When using an interface like
+   * \Drupal\Component\Plugin\ConfigurableInterface, this is where the
+   * configuration should be stored.
+   *
+   * Plugin configuration is optional, so plugin implementations must provide
+   * their own setters and getters.
+   *
+   * @var array
+   */
+  protected $configuration;
+
+  /**
+   * The breadcrumb builder object.
+   *
+   * @var \Drupal\ezcontent_api\Breadcrumb\EzContentBreadcrumbBuilder
+   */
+  protected $ezconteBreadcrumbBuilder;
+
+  /**
+   * The RouteMatchInterface object.
+   *
+   * @var \Drupal\Core\Routing\RouteMatchInterface
+   *   The current route match.
+   */
+  protected $routeMatch;
+
+  /**
+   * Constructs a \Drupal\Component\Plugin\PluginBase object.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\ezcontent_api\Breadcrumb\EzContentBreadcrumbBuilder $ezconteBreadcrumbBuilder
+   *   The breadcrumb builder object.*.
+   * @param \Drupal\Core\Routing\RouteMatchInterface $routeMatch
+   *   The RouteMatchInterface object.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EzContentBreadcrumbBuilder $ezconteBreadcrumbBuilder, RouteMatchInterface $routeMatch) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->ezconteBreadcrumbBuilder = $ezconteBreadcrumbBuilder;
+    $this->routeMatch = $routeMatch;
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('ezcontent_api_normalizer.ezcontent_breadcrumb'),
+      $container->get('current_route_match')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLink($resource_object) {
+    $entity = $this->routeMatch->getParameter('entity');
+    $link_attributes = [];
```

Comment:
Shall we use camelCases here ?
```suggestion
    $linkAttributes = [];
```